WTG.Analyzers.Utils.nuspec

```diff
@@ -18,5 +18,8 @@
 	<files>
 		<!-- Binaries -->
 		<file src="bin\netstandard1.3\WTG.Analyzers.Utils.dll" target="lib\netstandard1.3" />
+
+				<!-- Other -->
```

Comment:
The OCD in me says to normalise the indentation. :)

Reply:
Fixed here 265af8f8d5f02f0537189b4c8b4dfe3763a4feec